x-pack/packetbeat/cmd/agent.go

```diff
@@ -0,0 +1,24 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/elastic/beats/v7/libbeat/common/reload"
+	"github.com/elastic/elastic-agent-client/v7/pkg/client"
+	"github.com/elastic/elastic-agent-client/v7/pkg/proto"
+	conf "github.com/elastic/elastic-agent-libs/config"
+)
+
+// packetbeatCfg is a callback registered with central management to perform any needed config transformations
+// before agent configs are sent to a beat
```

Comment:
Would it be true to say the following
```
// packetbeatCfg returns a packetbeat Elastic Agent client.Unit reload
// configuration constructed from configuration details in rawIn.
```
in conjunction with a comment on the `SetTransform` call?

As a maintainer doing investigation into issues this is approximately the level of detail and direction in doc comments that helps focus or exclude parts of code to read.

Reply:
Yah, that would work too, I can expand the doc to sort of merge the two. Will also expand the `SetTransform` Docs a bit, since that's a bit vague.